custom_components/healthchecksio/config_flow.py

```
@@ -1,14 +1,24 @@
 """Adds config flow for HealthChecks.io"""
+from __future__ import annotations
+
+import asyncio
+import json
+from collections import OrderedDict
```

Ruff (0.11.9)

`collections.OrderedDict` imported but unused

Remove unused import: `collections.OrderedDict`

(F401)